_pages/covid19.html

```diff
@@ -0,0 +1,43 @@
+---
+description: DjangoCon US is committed to providing a safe in-person conference through the use of vaccine verification and a mask mandate.
+heading: COVID-19 In-Person Policy
+layout: base
+permalink: /covid/
+title: COVID-19 In-Person Policy
+---
+
+<header class="subpage-header">
+  <h1>{{ page.heading }}</h1>
+</header>
+
+<div class="content subpage-content">
+  <section class="section-pad">
+    <div class="row column">
+        <h2 class="demp">
+        DjangoCon US is committed to providing a safe in-person conference through the use of vaccine verification and a mask mandate.
+        </h2>
+    </div>
+  </section>
+
+  <section class="section-pad theme-light-gray">
+    <div class="row card-row">
+        <div class="medium-6 column">
+            <p>
+              In order to keep conference attendees safe, DjangoCon US will have an in-door mask mandate along with a vaccine verification process in-person in San Diego as well as before the conference.
+              <br />
+              <br />
+              For vaccine verification, we will be using a third party vendor recognized as a leader in the space. In order to pass the vaccine verification, you must have be up to date on your vaccine (including eligible boosters). We will be sending out the instructions to verify your vaccine status prior to the conference.
+              <br />
+              <br />
+              The conference will provide KN-95 / N-95 masks for people to wear. We will be asking that all attendees wear a certified KN-95 / N-95 mask while inside. 
```

Comment:
```suggestion
              The conference will provide KN95 and/or N95 masks for people to wear. We will be asking that all attendees wear a certified KN95 / N95 mask while inside. 
```
I'm not certain on this one, but I believe they're not hyphenated.